package/base-files/Makefile

```diff
@@ -43,7 +43,8 @@ define Package/base-files
 	+netifd +libc +jsonfilter +SIGNED_PACKAGES:usign +SIGNED_PACKAGES:openwrt-keyring \
 	+NAND_SUPPORT:ubi-utils +fstools +fwtool \
 	+SELINUX:procd-selinux +!SELINUX:procd +SECCOMP:procd-seccomp \
-	+SELINUX:busybox-selinux +!SELINUX:busybox
+	+SELINUX:busybox-selinux +!SELINUX:busybox +!SMALL_FLASH:procd-ujail \
+	+USE_APK:apk +!USE_APK:opkg
```

Comment:
FYI the `USE_APK:apk` is intentional here, otherwise `apk-openssl` can't be selected. I've tested this in `ImageBuilder` and it works with `apk-mbedtls` as expected.

```shell
$ make info
Current Target: "mediatek/filogic"
Current Architecture: "aarch64"
Current Revision: "r28105+3-afffcd09e5f1"
Default Packages: base-files ca-bundle dropbear fstools libc libgcc libustream-mbedtls logd mtd netifd uci uclient-fetch urandom-seed urngd kmod-leds-gpio kmod-gpio-button-hotplug fitblk kmod-phy-aquantia kmod-crypto-hw-safexcel wpad-basic-mbedtls uboot-envtools dnsmasq firewall4 nftables kmod-nft-offload odhcp6c odhcpd-ipv6only ppp ppp-mod-pppoe
```

```shell
./staging_dir/host/bin/apk adbdump packages/base-files-1638~b875e1728025.apk
#%SCHEMA: 676B6370
# ADB block, size: 11668, compat: 0, ver: 0
info:
  name: base-files
  version: 1638~b875e1728025
  hashes: d7e528067719fcb04701ddf3b28486ea1b38af81
  description: |
    This package contains a base filesystem and system scripts for OpenWrt.
  arch: aarch64_cortex-a53
  license: GPL-2.0
  origin: feeds/base/base-files
  url: http://openwrt.org/
  installed-size: 556 KiB
  depends: # 13 items
    - apk
    - busybox
    - fstools
    - fwtool
    - jsonfilter
    - libc
    - netifd
    - openwrt-keyring
    - procd
    - procd-seccomp
    - procd-ujail
    - ubi-utils
    - usign
```